solutions/usaco-531.mdx

```diff
@@ -122,3 +122,66 @@ int main()
 </CPPSection>
 
 </LanguageSection>
+
+
+## Prim's Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+	
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+	
+public class Superbull {
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio("superbull");
+		int N = io.nextInt();
+		// Read in the input array.
+		int[] ids = new int[N];
+		for (int i = 0; i < N; i++) {
+			ids[i] = io.nextInt();
+		}
+
+		long max_cost = 0;
+		boolean[] visited = new boolean[N];
+		int[] score = new int[N];
+
+		// Loop through possible choices for the first team.
+		for (int team1 = 0; team1 < N; team1++) {
```

Comment:
:/ I don't think prim
> Loop through possible choices for the first team.

Doesn't it loop through N iterations, keeping track of node with the minimum (or maximum in this case) weight? It doesn't have anything to do with `team1`?